web/.htaccess

```diff
@@ -13,11 +13,18 @@
 <IfModule mod_expires.c>
     ExpiresActive on
     ExpiresDefault                        "access plus 1 week"
+
     ExpiresByType application/json        "access plus 0 seconds"
-    ExpiresByType application/javascript  "access plus 1 year"
-    ExpiresByType application/font-woff   "access plus 1 month"
     ExpiresByType image/gif               "access plus 1 month"
     ExpiresByType image/jpeg              "access plus 1 month"
     ExpiresByType image/png               "access plus 1 month"
     ExpiresByType text/css                "access plus 1 year"
+    ExpiresByType application/font-woff   "access plus 1 year"
+    ExpiresByType application/javascript  "access plus 1 year"
+</IfModule>
+
+<IfModule mod_deflate.c>
+  <FilesMatch "\.(woff|svg)$" >
+    SetOutputFilter DEFLATE
```

Comment:
I confess I'm not familiar with this directive, but I wonder if it could be written like this (based on [this page](https://davidwalsh.name/html5-boilerplate-htaccess))
```
AddOutputFilterByType DEFLATE application/font-woff \
                              image/svg+xml
```